layers_ajax.php

```diff
@@ -72,6 +72,11 @@
       'source' => $layer['cal_layeruser'],
       'color' => $layer['cal_color'],
       'dups' => $layer['cal_dups'],
+// Craig. This looks like a bug. I can't find $layerfullname set anywhere.
+// And the only table with a fullname is webcal_entry_ext_user, which doesn't seem to be called here.
+// Even $fullname doesn't seem to fit.
+// Is it supposed to be maybe $layer['cal_login'] or $layer['cal_layeruser']?
+
       'fullname' => $layerfullname];
```

Comment:
Well, since it's not set anywhere I can find, I suggest this line be changed to
'fullname' => layer['cal_layeruser'];